scripts/exrrfs_jedivar.sh

```diff
@@ -54,7 +57,32 @@ cp "${UMBRELLA_PREP_IC_DATA}"/*satbias* satbias_in/.
 #
 # copy observations files
 #
-cp "${COMOUT}"/ioda_bufr/det/* obs/.
+obspath_bufr="${COMOUT}/ioda_bufr/${WGF}"
+obs_files_source[0]="${obspath_bufr}"/ioda_adpupa.nc
+obs_files_target[0]=ioda_adpupa.nc
+
+obs_number=${#obs_files_source[@]}
+obs_files_source[obs_number]="${obspath_bufr}"/ioda_aircar.nc
+obs_files_target[obs_number]=ioda_aircar.nc
+
+if [ "${DO_ENVAR_RADAR_REF}" == "true" ];then
+  obspath_refl="${COMOUT}/ioda_mrms_refl/${WGF}"
+  obs_number=${#obs_files_source[@]}
+  obs_files_source[obs_number]="${obspath_refl}/ioda_mrms_${CDATE}_${time_min}".nc4
+  obs_files_target[obs_number]=ioda_mrms.nc
+fi
+
+obs_number=${#obs_files_source[@]}
+for (( i=0; i<obs_number; i++ ));
+do
+  obs_file=${obs_files_source[$i]}
+  obs_file_t=${obs_files_target[$i]}
+  if [ -r "${obs_file}" ]; then
+    cp "${obs_file}" obs/"${obs_file_t}"
+  else
+    echo "WARNING: ${obs_file} does not exist!"
+  fi
+done
```

Comment:
Lines 60-85:
I am thinking of whether we still need this logic. I knew we used it in RAP/HRRR/RRFS.v1. But RRFS.v2 has extra ioda converter steps, we can check observation availability there if needed. Also, JEDI itself handles missing ioda observations as well. If possible, I would suggest we simply copy ioda files as follows:
```
cp "${COMOUT}"/ioda_bufr/det/* obs/.
cp "${COMOUT}"/ioda_mrms_ref/det/* obs/.
```
Thanks!

Reply:
I think we need to control what used and avoid using any "*".

Reply:
since Lines 57-86 will expand more and more when we add more observations, I would suggest we put this part into a separate script, such as "copy_obs.sh",  and put it under `USHrrfs`.  We then `source ${USHrrfs}/copy_obs.sh` here.  And we can reuse this script fot GETKF.